### ISSUE TITLE
instead of using vgg try using resnet it will give good accuracy on the valid data set

### ISSUE THREAD

**Issue by 27shree23** [146790g/BreakHis-VGG16#1]

**Comment by 146790g** [comment]
thank you so much,  I will use with ResNet50 . 
after I complete, I will upload the result on my Github page.  see you later.

**Comment by 146790g** [comment]
next i apply VGG19 model on Breakhis, but i encounter error. would you mind telling me how to solve ??

****************************************************************************************************************************
WARNING:tensorflow:From <ipython-input-12-1ec9f76af414>:7: Model.fit_generator (from tensorflow.python.keras.engine.training) is deprecated and will be removed in a future version.
Instructions for updating:
Please use Model.fit, which supports generators.
Epoch 1/50
 2/44 [>.............................] - ETA: 27s - loss: 1.5824 - accuracy: 0.5395WARNING:tensorflow:Callbacks method `on_train_batch_end` is slow compared to the batch time (batch time: 0.4162s vs `on_train_batch_end` time: 0.8725s). Check your callbacks.
44/44 [==============================] - 56s 1s/step - loss: 0.6690 - accuracy: 0.6782 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00
Epoch 2/50
44/44 [==============================] - 57s 1s/step - loss: 0.5539 - accuracy: 0.7111 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00
Epoch 3/50
44/44 [==============================] - 57s 1s/step - loss: 0.4926 - accuracy: 0.7907 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00
Epoch 4/50
44/44 [==============================] - 57s 1s/step - loss: 0.5209 - accuracy: 0.7781 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+00
Epoch 5/50
44/44 [==============================] - 57s 1s/step - loss: 0.4832 - accuracy: 0.7984 - val_loss: 0.0000e+00 - val_accuracy: 0.0000e+0
